app/src/main/java/app/insti/fragment/ComplaintsHomeFragment.java

```diff
@@ -47,8 +47,8 @@ public void onStart() {
         swipeContainer.post(new Runnable() {
             @Override
             public void run() {
+                callServerToGetNearbyComplaints();
                 if (complaints == null) {
-                    callServerToGetNearbyComplaints();
```

Comment:
Why this change?

Reply:
Once you navigate to a complaint details fragment from the complaints home or me page, on the back press the complaints are not updated. So if a user upvotes or comments on the complaint it won't be reflected on the back press.
So this change refreshes the complaints every time the page is loaded.

Reply:
This change was made deliberately in 6f6736df814818a294ea64b306135711f449b730 since loading complaints every time the user navigates back to complaints is:
1) Excessive unnecessary server calls. Especially with paging, if the user has loaded 50 complaints, then this is effectively 5 unnecessary expensive API calls if the user ever does scroll down to the complaint (it doesn't matter anyway if the user doesn't)
2) Terrible UX, for obvious reasons

There are two ways around this:
1) Use a global static memcache (similar to `UpdatableList`s in Utils for events, bodies, notifications etc.) and update this as things change.
2) Don't fix it, since the tradeoff of UX is too much comparable to the negligible gain that we are getting here.

Reply:
Alright I've reverted the change